packages/imperative/src/cmd/src/yargs/YargsConfigurer.ts

```diff
@@ -40,6 +40,11 @@ export class YargsConfigurer {
     ) {
     }
 
+    public static yargsConfiguration: Record<string, boolean> = {
```

Comment:
SonarCloud is suggesting to define this as `public static const` or `public static readonly`.

Reply:
+1 for `public static readonly` as `public static const` doesn't work in TypeScript 😋 

Although `readonly` will prevent us from changing the entire record, you can still technically update keys inside it. Since the parsing configuration is only defined once and isn't modified after initialization, we could wrap the type in `Readonly`, and TypeScript will emit an error when any changes are made to the record:

```suggestion
    public static yargsConfiguration: Readonly<Record<string, boolean>> = {
```

![image](https://github.com/user-attachments/assets/92da025d-8090-4e92-b6cc-93e284d2464d)